conseil-api/src/main/resources/application.conf

```diff
@@ -11,10 +15,18 @@ conseil {
   # Replace database name, user and password in an env-specific config file.
   db {
     dataSourceClass: "org.postgresql.ds.PGSimpleDataSource"
+    dataSourceClass: ${?CONSEIL_API_DB_SOURCE_CLASS}
```

Comment:
To be honest, I doubt it would work with something different, but let's leave it like this... maybe in the future?

Reply:
You are right, but either way, we are giving this option (to change that) no matter if I'll expose it like that or now. You can always attach your own configuration file and change that. I'm just following the convention (at least for most of the properties). If you think it should be "harder" to overwrite that, we can remove it, not a problem. Maybe even we should to:

a) expose only these fields, we want to be easily changed
b) remove macros for fields, we don't want so easy to be changed (potential user may ask us, why it's not exposed and we can explain that). 

Which option would you like to follow? I like b) a bit more tho - meaning, I'll remove it.

Reply:
as I mentioned, I'd lean for option `b`, to remove it